lib/kubeclient.rb

```diff
@@ -449,6 +469,32 @@ def delete_entity(resource_name, name, namespace = nil, delete_options: {})
       format_response(@as, response.body)
     end
 
+    # Accepts the following options:
+    #   :namespace (string) - the namespace of the entity.
+    #   :label_selector (string) - a selector to restrict the list of returned objects by labels.
+    #   :field_selector (string) - a selector to restrict the list of returned objects by fields.
+    #   :grace_period_seconds (integer) - the duration before an object should be deleted
+    #   :limit (integer) - a maximum number of items to return in each response
+    #   :orphan_dependents (bool) - should the dependent objects be orphaned
+    #   :propagation_policy (string) - one of Foreground|Background|Orphan
+    #   :resource_version (string) - sets a limit on the resource versions that can be served
+    #   :resource_version_match (string) - determines how the resource_version constraint will be applied
```

Comment:
lib/kubeclient.rb:481:101: C: Layout/LineLength: Line is too long. [105/100]

Opened #547 to add support `resource_version_match` in other actions.